data_steward/cdr_cleaner/cleaning_rules/remove_ehr_data_past_deactivation_date.py

```diff
@@ -58,7 +57,7 @@ def remove_ehr_data_queries(project_id, ticket_number, pids_project_id,
     rdp.add_console_logging(ARGS.console_log)
 
     remove_ehr_data_queries = remove_ehr_data_queries(ARGS.project_id,
-                                                      ARGS.ticket_number,
+                                                      TICKET_NUMBER,
```

Comment:
I think I understand your suggested change to `--ticket_number` in `retract_deactivated_pids.py`.  It doesn't really make a lot of sense to have to provide it here and not use what is provided.  But, to avoid breaking that, you either have to revert these arg_parse changes or add in a safety check in `retract_deactivated_pids` that bails out if the ticket number is not defined.